ui/src/components/songsView/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Container, Box } from '@mui/material';
```

Comment:
I would advice against naming this index.tsx, it would make it hard to find in the future. maybe SongsView is more appropriate?

Reply:
changed to `SongsViewContainer`